### ISSUE TITLE
Fine-Tune again

### ISSUE THREAD

**Issue by M-106** [xXAI-botXx/Calm-down-bot#13]
- Use maybe your computer
- Remove the padding
- Set max-length to 1024
- Padding have to be on the right
- Trim the data in dataloder
- Try out with label as bot answer?
- Test out/improve attention layer
    - 0=Train, 1=Target

**Comment by M-106** [comment]
Use Tensorboard https://pytorch.org/tutorials/recipes/recipes/tensorboard_with_pytorch.html

**Comment by M-106** [comment]
Auch die Antworten mit drin lassen

**Comment by M-106** [comment]
Run the training on your local computer
